--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-project = 'YosysHQ Docs'
+project = 'YosysHQ Unnamed Project'
 author = 'YosysHQ GmbH'
 copyright = '2024 YosysHQ GmbH'
 
@@ -8,10 +8,6 @@ html_theme = 'furo-ys'
 html_css_files = ['custom.css']
 html_theme_options: dict[str, any] = {}
 
-# this is the root doc, we don't need to provde links back here
-html_theme_options["brand_main_docs"] = None
-html_theme_options["brand_main_search"] = None
-
 # These folders are copied to the documentation's HTML output
 html_static_path = ['_static']
